includes/admin/hooks/class-wc-woomercadopago-hook-order-details.php

```diff
@@ -0,0 +1,232 @@
+			'pending_card_validation' => 'pending',
+			'pending_online_validation' => 'pending',
+			'pending_additional_info' => 'pending',
+			'offline_process' => 'pending',
+			'pending_challenge' => 'pending',
+			'pending_provider_response' => 'pending',
+			'bank_rejected' => 'rejected',
+			'rejected_by_bank' => 'rejected',
+			'rejected_insufficient_data' => 'rejected',
+			'bank_error' => 'rejected',
+			'by_admin' => 'rejected',
+			'expired' => 'rejected',
+			'cc_rejected_bad_filled_card_number' => 'rejected',
+			'cc_rejected_bad_filled_security_code' => 'rejected',
+			'cc_rejected_bad_filled_date' => 'rejected',
+			'cc_rejected_high_risk' => 'rejected',
+			'cc_rejected_fraud' => 'rejected',
+			'cc_rejected_blacklist' => 'rejected',
+			'cc_rejected_insufficient_amount' => 'rejected',
+			'cc_rejected_other_reason' => 'rejected',
+			'cc_rejected_max_attempts' => 'rejected',
+			'cc_rejected_invalid_installments' => 'rejected',
+			'cc_rejected_call_for_authorize' => 'rejected',
+			'cc_rejected_duplicated_payment' => 'rejected',
+			'cc_rejected_card_disabled' => 'rejected',
+			'payer_unavailable' => 'rejected',
+			'rejected_high_risk' => 'rejected',
+			'rejected_by_regulations' => 'rejected',
+			'rejected_cap_exceeded' => 'rejected',
+			'cc_rejected_3ds_challenge' => 'rejected',
+			'rejected_other_reason' => 'rejected',
+			'authorization_revoked' => 'rejected',
+			'cc_amount_rate_limit_exceeded' => 'rejected',
+			'cc_rejected_expired_operation' => 'rejected',
+			'cc_rejected_bad_filled_other' => 'rejected',
+			'rejected_call_for_authorize' => 'rejected',
+			'am_insufficient_amount' => 'rejected',
+			'generic' => 'rejected',
+		];
+		$status           = array_key_exists($order_status, $processed_status) ? $processed_status[$order_status] : $processed_status['generic'];
+
+		return $status;
+	}
+
+	/**
+	 * Create payment status metabox
+	 *
+	 * @return void
+	 */
+	public function payment_status_metabox( $screen_name ) {
+		add_meta_box(
+			'payment-status-metabox',
+			__( 'Payment status on Mercado Pago', 'woocommerce-mercadopago' ),
+			[$this, 'payment_status_metabox_content'],
+			$screen_name
+		);
+	}
+
+	/**
+	 * Payment Status Metabox Script
+	 *
+	 * @return void
+	 */
+	public function payment_status_metabox_script() {
+		$suffix = $this->get_suffix();
+
+		wp_enqueue_script(
+			'payment_status_metabox',
+			plugins_url( '../../assets/js/payment_status_metabox' . $suffix . '.js', plugin_dir_path( __FILE__ ) ),
+			array(),
+			WC_WooMercadoPago_Constants::VERSION,
+			false
+		);
+	}
+
+	/**
+	 * Payment Status Metabox Content
+	 *
+	 * @return void
+	 */
+	public function payment_status_metabox_content() {
+		$order_id     = ! empty($_GET['post']) && ! wp_verify_nonce(sanitize_text_field($_GET['post'])) ? sanitize_text_field($_GET['post']) : null;
+		$order        = ! is_null($order_id) ? wc_get_order($order_id) : null;
+		$alert_status = $this->get_processed_status($order->get_status());
```

Comment:
Se por um acaso o $order for null, ele não vai ter o método get_status()

E também verificar se é um pagamento do Mercado Pago, para garantir que não apareça se não for de Mercado Pago

No WC_Order tem um método get_payment_method() ai vem com o ID do gateway